src/pymovements/dataset/dataset.py

```diff
@@ -1005,6 +1010,7 @@ def extract(
             self,
             remove_finished: bool = False,
```

Comment:
can you add a `*,` before `remove_finished`? This is PR a breaking change either way, so we can make it more future proof.